### PR TITLE
Add stockNumber #1670

### PR DESCRIPTION
**Pull request by TobiasNx**
Resolves: #1670

Please have a look.
I introduced stockNumber.

I was not sure if I should model this as an array of Objects to state the sources of the stockNumber too. Or just like this.

### CODE REVIEW

**Comment by dr0i** [comment]
Issue is missing, I don't know the context. You modelled this field at root level. Is this OK? What is the difference to `callnumber` (which is modelled under the `item` subtree)?

**Comment by TobiasNx** [comment]
> Issue is missing, I don't know the context. You modelled this field at root level. Is this OK? What is the difference to `callnumber` (which is modelled under the `item` subtree)?

This #1670

**Comment by dr0i** [comment]
Thx for the reference.  Have you considered to put this field under `publication`?

**Review by dr0i — CHANGES_REQUESTED** [review]

File: src/main/resources/labels/context-labels.json [hunk 1]
```diff
@@ -747,5 +747,11 @@
     "multilangLabel": {},
     "name": "bibliographicLevel",
     "uri": "http://purl.org/lobid/lv#bibliographicLevel"
+  },
+  {
+    "label": "Bestellnummer",
+    "multilangLabel": {},
+    "name": "stockNumber",
+    "uri": "http://purl.org/lobid/lv#stockNumber"
```
[the remark this answers is not in the document]
Comment (TobiasNx) [reply src/main/resources/labels/context-labels.json:755]: Opend a PR now: https://github.com/hbz/lobid-vocabs/pull/178

[the remark this answers is not in the document]
Comment (TobiasNx) [reply src/main/resources/labels/context-labels.json:755]: Since it is part of lobid vocabs now. Done.

**Pull request merged**